build.gradle.kts

```diff
@@ -54,6 +62,56 @@ subprojects {
     version = LIBRARY_VERSION
 }
 
-tasks.register<Delete>("clean") {
+tasks.getByName<Delete>("clean") {
     delete(rootProject.layout.buildDirectory)
 }
+
+// Merges individual module docs into a single HTML output
+dependencies {
+    dokka(project(":core:"))
+    dokka(project(":connectors:supabase"))
```

Comment:
I assume this should also include `:compose`?

Reply:
Good point! I forgot that module existed 😁 . Added it now.